src/pages/TeamSettings/Service-Accounts.vue

```diff
@@ -78,7 +78,7 @@ export default {
         })
         .catch(({ graphQLErrors }) => {
           this.accountCreationError = `There was an error creating your service account:
-            ${{ graphQLErrors }}`
+            ${graphQLErrors[0].message}`
```

Comment:
Do we know there will always be a message at the first index here or should this be?
```suggestion
            ${graphQLErrors?.[0].message}`
```

Reply:
I think there should likely be, but better to be safe.